### PR TITLE
Add page template meta tags

### PR DESCRIPTION
**Pull request by dhmacs**
closes https://github.com/iFixit/ifixit/issues/50536

## QA

1. Open [Strapi Govinor preview](https://add-page-template-metatags.govinor.com/admin/content-manager/collectionType/api::page.page/2?plugins[i18n][locale]=en)
2. Change store home page meta title and description
3. Verify that the [store home page preview](https://react-commerce-git-add-page-template-metatags-ifixit.vercel.app/Store) has page title and meta tags (title, description and `og`) that reflect what's been set in Strapi

### CODE REVIEW

File: frontend/templates/page/meta-tags.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import type { Page } from '@models/page';
+import Head from 'next/head';
+
+interface MetaTagsProps {
+   page: Page;
+}
+
+export function MetaTags({ page }: MetaTagsProps) {
+   const ogImage = findOgImage(page);
+   console.log('ogImage', ogImage);
```
Comment (masonmcelvain) [inline frontend/templates/page/meta-tags.tsx:13]:
```suggestion
```

**Review by masonmcelvain — DISMISSED** [review]
CR 👍🏻

**Review by masonmcelvain — APPROVED** [review]
CR 👏🏻

**Comment by deltuh-vee** [comment]
QA 🎬 
Store page now has page title and meta tags that reflect what's been set in Strapi

**Pull request merged**